rental.rb

```diff
@@ -0,0 +1,15 @@
+# recored of rented books
+class Rental
+  attr_reader :person, :book
```

Comment:
- [ ] I would implore you to make this also have setter since we would be setting the data too.